node/async-api-converter/README.md

```diff
@@ -0,0 +1,70 @@
+
+-   Supporting the `ws` (WebSockets) protocol:  
+    AsyncAPI treats WebSocket connections as independent protocol with given restrictions, e.g., only one websocket connection is possible per AsyncAPI instance (because routing channels to certain servers is not possible and WebSocket does not support channels/topics/...). This should be taken into account for the conversion, one promising approach for converting protocols other then mqtt-like, could be to generate one AsyncAPI instance per available server to deal with the in this case more expressive TDs.
+-   Adding security support:  
+    The focus of AsyncAPI is hardly on security (since event-based solutions are often used in closed systems) but still it supports quite some mechanisms. The more complicated part in converting this from TD to AsyncAPI is that TDs support different security mechanisms for every operation, while AsyncAPI allows only defining them per Server (and there is no server to channel/operation/message mapping).
+
+-   Add further use cases:  
+    Currently only two types of TD interactions are mapped to their AsyncAPI equivalents:
+
+    -   Properties with the op `observeproperty` indicating that the property can be watched asynchronously
+    -   Events to which a client can subscribe (`subscribeevent`)
+
+    But there could be other use cases that should be taken into account, e.g., reading (possible with retain option) variables via mqtt that might not be an asynchronous operation itself but useful to support since `mqtt` operations cannot be represented with OpenAPI instances.
+
+-   Think about how to treat interactions with the same path (-> same AsyncAPI channel), e.g., a property and an event that both interact with `thing.example.com/status`
+
+## Comments
+
+A TD hasn't necessarily to be valid in order to be converted to an AsyncAPI instance. This converter will only throw an Error if the invalid part has a strong effect on the conversion result, but tries to ignore the most cases. This is by purpose, since there is the possibility to validate a TD using, e.g., the playground core package and the conversion of experimental TDs for example to create new TD features, should be supported.
+
+## Usage
+
+You can use this package to integrate AsyncAPI instance generation from a TD in your application.
+
+-   Install this package via NPM (`npm install @thing-description-playground/td_to_asyncapi`) (or clone repo and install the package with `npm install`)
+-   Node.js or Browser import:
+
+    -   Node.js: Require the package and use the functions
+
+    ```javascript
+    const tdToAsyncAPI = require("@thing-description-playground/td_to_asyncapi");
```

Comment:
I think these require statements needs to change as well. There are other below. At the moment it is a bit fuzzy since we do not have a publication. maybe once merged we should publish a new release soon.